### DISCUSSION TITLE
Emulator list

### DISCUSSION THREAD

**Discussion by HunterE1437** [EmulatorJS/EmulatorJS#472] [General]
I'm trying to make a list of game emulators (snes, nes , game boy) for educational purposes ( I don't want to do work at school). And I figured this was the best place to find them, but I cant find many good browser emulators, i was wondering if anyone has any that would work on a Chromebooks Google browser?

**Comment by ethanaobrien** [comment]
Well theres this: http://raw.githack.com/EmulatorJS/emulator-button/main/index.html
and this: https://emulatorjs.github.io/emulator-button/index.html
and (if all else blocked) this: https://canvas-cluster-009.herokuapp.com/https://ethanaobrien.github.io/emulator-button/index.html

There is also the main demo: https://demo.emulatorjs.org/

I am the author of all above mentioned projects. Ask me if you have any questions!
